classes/classes/Scenes/Areas/Forest/KitsuneScene.as

```diff
@@ -90,7 +90,7 @@ package classes.Scenes.Areas.Forest
 				doNext(createCallBackFunction(followTheWillOWisp, true));
 			}//PC did NOT see through glamour
 			//With Religious BG:
-			else if (player.findPerk(PerkLib.HistoryReligious) >= 0) {
+			else if (player.findPerk(PerkLib.HistoryReligious) >= 0 || player.findPerk(PerkLib.EnlightenedNinetails) >= 0 || player.findPerk(PerkLib.CorruptedNinetails) >= 0) {
```

Comment:
Usually a better idea to split different changes into a different commit, i.e. group related changes into commits.